dev/doc/README.md

```diff
@@ -6,6 +6,40 @@ See the first section of [`INSTALL`](../../INSTALL).  Developers are
 recommended to use a recent OCaml version, which they can get through
 opam or Nix, in particular.
 
+## Configuring Dune caching and default verbosity
+
+There are several configuration settings that you can control globally
+with a Dune configuration file (see `man dune-config` to learn
+more). This file is generally located in `~/.config/dune/config` and
+should start with the version of the Dune language used, e.g.:
+
+```
+(lang dune 2.0)
+```
+
+- You will get faster rebuilds if you enable Dune caching. This is
+  true in all cases, but even more so when using the targets in
+  `Makefile.dune` (see below).
+
+  To set up Dune caching, you should append the following line to your
+  Dune configuration file:
+
+  ```
+  (cache enabled)
+  ```
+
+  Note that by default, Dune caching will use up to 10GB of disk size.
+  See the [official documentation](https://dune.readthedocs.io/en/stable/caching.html#on-disk-size)
+  to learn how to change the default.
+
+- Dune is not very verbose by default. If you want to change the
+  behavior to a more verbose one, you may append the following line to
+  your Dune configuration file:
+
+  ```
+  (display short)
+  ```
+
```

Comment:
FWIW the Dune doc doesn't mention how to put a comment in the file.  What is the convention?

Reply:
LISP syntax, that is to say `; comment`